### PR TITLE
Log warning on missing inspection data

### LINKED ISSUE #336: Empty inspection list is not handled in storage handler
**Describe the bug**
When the monitor step attempts to retrieve inspections for a given step in a mission to put them in the upload queue, and it gets no inspections, there are no warnings being triggered.
The relevant code is found in [monitor.py](https://github.com/equinor/isar/tree/main/src/isar/state_machine/states/monitor.py) line 79.

**To Reproduce**
Run a mission in which the inspection data is faulty.

**Expected behavior**
We expect a logged warning when there is no inspection data where we expected there to be some.

**Screenshots**

### PR DESCRIPTION
**Pull request by tsundvoll**
Fixes #336.

### CODE REVIEW

**Review by oysand — APPROVED** [review]
LGTM

File: src/isar/state_machine/states/monitor.py [hunk 1]
```diff
@@ -85,6 +85,11 @@ def _queue_inspections_for_upload(self, current_step: InspectionStep):
             )
             return
 
+        if len(inspections) == 0:
```
Comment (aestene) [inline src/isar/state_machine/states/monitor.py:88]:
```suggestion
        if not inspections:
```

Comment (aestene) [reply src/isar/state_machine/states/monitor.py:88]: More the "python" way of doing it I think.

**Review by aestene — APPROVED** [review]
One comment you may fix otherwise good.

File: src/isar/state_machine/states/monitor.py (hunk 1 above)
Comment (tsundvoll) [reply src/isar/state_machine/states/monitor.py]: Good point, fixed.

**Pull request merged**